examples/golang/topic-creation/README.md

```diff
@@ -0,0 +1,12 @@
+# Introduction
+This directory contains Golang code example on how to create an SNS Topic on a Ceph RGW cluster.
+
+# Prerequisite
+Linux machine running an RGW Ceph cluster. Preferrably started with the ``OSD=1 MON=1 MDS=0 MGR=0 RGW=1 ../src/vstart.sh --debug --new `` command.  
+Go installed on the Linux machine.  
+
+## Workflow Procedure
+1. Navigate through your file system to where the Golang example code exists on your terminal.
+2. Run ``` go mod tidy ``` to install the required Golang packages on the system.
+3. Run the Golang program as ``` go run topic-creation.go -t sample-topic ``` on the terminal window to create an SNS Topic on the Ceph RGW cluster.   
```

Comment:
please remove (3). creating a topic without an endpoint is used only in a deprecated mode called "pubsub".
lets not provide an example for that